### PR TITLE
[MODULAR] Makes the Poncho Greyscale-customizable!

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
This is totally not to fulfill my own dark needs
image
image

The trim is a seperate color value, so you can set it to slightly-darker than the base color to essentially hide it. It was that or have a whole lot more .json stuff for the single item, so

Also adds a toggled obj sprite, so it doesnt turn invisible. IDK how to fix it to update the mob sprite on-toggle tho, so it needs to be taken off and put back on still... issue for another PR!

## How This Contributes To The Skyrat Roleplay Experience
Customization hnngh

## Changelog

:cl:
add: The Ranger Poncho (brown cowboy poncho) is now Greyscale-able! (The default values are the same as the old sprite, so you dont NEED to customize it)
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/customization/modules/clothing/neck/cloaks.dm [hunk 1]
```diff
@@ -28,15 +28,18 @@
...
+/obj/item/clothing/neck/ranger_poncho
+	name = "ranger poncho"
 	desc = "Aim for the Heart, Ramon."
-	icon = 'modular_skyrat/master_files/icons/obj/clothing/neck.dmi'
-	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/neck.dmi'
-	icon_state = "ranger_cloak"
+	icon_state = "ranger_poncho"
+	greyscale_config = /datum/greyscale_config/ranger_poncho
+	greyscale_config_worn = /datum/greyscale_config/ranger_poncho/worn
+	greyscale_colors = "#917A57,#858585"	//Roughly the same color as the original non-greyscale item was
```
Comment (ghost) [inline modular_skyrat/modules/customization/modules/clothing/neck/cloaks.dm]:
```suggestion
	greyscale_colors = "#917A57#858585"	//Roughly the same color as the original non-greyscale item was
```
no comma, dumbass

File: modular_skyrat/modules/loadouts/loadout_items/loadout_datum_neck.dm [hunk 2]
```diff
@@ -132,9 +132,9 @@ GLOBAL_LIST_INIT(loadout_necks, generate_loadout_items(/datum/loadout_item/neck)
 	name = "Green cowboy poncho"
 	item_path = /obj/item/clothing/neck/cowboylea
 
-/datum/loadout_item/neck/brownponchocowboy
-	name = "Brown cowboy poncho"
-	item_path = /obj/item/clothing/neck/ponchoranger
+/datum/loadout_item/neck/poncho_ranger
```
Comment (ghost) [inline modular_skyrat/modules/loadouts/loadout_items/loadout_datum_neck.dm]:
```suggestion
/datum/loadout_item/neck/ranger_poncho_greyscale
```
better datum name

File: modular_skyrat/modules/loadouts/loadout_items/loadout_datum_neck.dm [hunk 3]
```diff
@@ -132,9 +132,9 @@ GLOBAL_LIST_INIT(loadout_necks, generate_loadout_items(/datum/loadout_item/neck)
...
+	name = "Greyscale cowboy poncho"
```
Comment (ghost) [inline modular_skyrat/modules/loadouts/loadout_items/loadout_datum_neck.dm]:
```suggestion
	name = "Greyscale ranger poncho"
```

**Review by ghost — COMMENTED** [review]
hnngh

**Comment by ghost** [comment]
that SHOULD have solved the runtime. my silly poopoo brain added a comma to the color string :)

**Review by Gandalf2k15 — APPROVED** [review]

**Pull request merged**